prime-router/metadata/schemas/direct/imagemover-covid-19.schema

```diff
@@ -0,0 +1,16 @@
+---
+name: imagemover-covid-19
+description: ImageMover
+topic: covid-19
+trackingElement: specimen_id
+basedOn: covid-19
+extends: direct/direct-covid-19
+
+elements:
+  - name: sender_id
+    default: ImageMover
+
+  - name: processing_mode_code
+    default: T
```

Comment:
Right here is the reason for all the fuss with processing_mode_code :   I can default this to 'T' here in a sender's schema, then I *know* they can safely send junk data into Production.   Critically, this lets them test with PII data, which they like.  When we are ready, we update their schema to "P" and they are live.

I thought briefly about adding "Sender Defaults", in the settings, as another way to implememt this same idea, instead of the above child-schema-with-default.   But there are so many darn ways to default things already, it seemed wiser to use one our existing clever mechanisms for doing the same thing.